testing/compatibility/azure/install-tentacle.ps1

```diff
@@ -4,7 +4,13 @@ $octopusServerApiKey,
 $octopusServerRole,
 $octopusServerEnvironment,
 $os,
-$tentacleUri)
+$tentacleUri,
+$tentacleNamePostfix)
+
+
+myargs = "$octopusServerThumbprint $octopusServerUrl $octopusServerApiKey $octopusServerRole $octopusServerEnvironment $os $tentacleUri $tentacleNamePostfix"
```

Comment:
Nit: Maybe add a comment as to what this is doing and why it is here.

I helped you with some of the powershell, and I initially forgot... 😁

Reply:
Done